Containerfiles/centos7.Containerfile

```diff
@@ -9,10 +9,14 @@ ENV APP_DEV_DEPS "requirements/centos7.requirements.txt"
 
 WORKDIR /data
 
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-* && \
+    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=https://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*
```

Comment:
```suggestion
RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* && \
    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=https://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
```
There are no CentOS-Linux-* repofiles on CentOS Linux 7:
```
# ll /etc/yum.repos.d/
-rw-r--r--. 1 root root 1664 May 21 10:48 CentOS-Base.repo
-rw-r--r--. 1 root root 1309 May 21 10:48 CentOS-CR.repo
-rw-r--r--. 1 root root  649 May 21 10:48 CentOS-Debuginfo.repo
-rw-r--r--. 1 root root  314 May 21 10:48 CentOS-fasttrack.repo
-rw-r--r--. 1 root root  630 May 21 10:48 CentOS-Media.repo
-rw-r--r--. 1 root root 1331 May 21 10:48 CentOS-Sources.repo
-rw-r--r--. 1 root root 9454 May 21 10:48 CentOS-Vault.repo
-rw-r--r--. 1 root root  616 May 21 10:48 CentOS-x86_64-kernel.repo
```

Reply:
It's incorrect in https://github.com/oamg/convert2rhel/pull/1294 as well.

Reply:
😆 spreading the wrong code over all repos! Let me fix this for both repos